steps/genymotion-cloud-saas-start/0.1.0/step.yml

```diff
@@ -0,0 +1,91 @@
+  1. Navigate to Default templates or templates:
+    - Select a device.
+    - Click on dot menu and select details.
+    - On the description field, get the recipe UUID.
+    - Copy paste it to use it in this step.
+website: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-start
+source_code_url: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-start
+support_url: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-start/issues
+published_at: 2020-02-20T09:57:45.011317504+01:00
+source:
+  git: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-start.git
+  commit: 4f02e00ecad99c29f652981b9b3c6b9f2461055b
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- utility
+toolkit:
+  go:
+    package_name: github.com/genymobile/bitrise-step-genymotion-cloud-saas-start
+deps:
+  brew:
+  - name: python3
+  - bin_name: pip3
+  apt_get:
+  - name: python3-pip
+  - bin_name: pip3
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- email: ""
+  opts:
+    description: Email of your Genymotion Cloud SaaS account, if you don't have an
+      account please create it first on [https://cloud.geny.io](https://cloud.geny.io/?&utm_source=web-referral&utm_medium=docs&utm_campaign=bitrise&utm_content=signup)
+    is_required: true
+    summary: ""
+    title: Genymotion Cloud SaaS email
+- opts:
+    description: Password of your Genymotion Cloud SaaS account.
+    is_required: true
+    is_sensitive: true
+    summary: ""
+    title: Genymotion Cloud SaaS password
+  password: ""
+- opts:
+    description: "Recipe UUID is the identifier used when starting an instance, Recipe
+      UUID can be retrieved using `gmsaas recipes list`.\n\nFormat:\nYou can specify
+      several recipe UUIDs in order to start several instances. Recipe UUIDs are separated
+      with a comma.\n\nFor example:\n`e20da1a3-313c-434a-9d43-7268b12fee08,c52fdfc2-6914-4266-aa6e-50258f50ef91,06867de4-4b99-4842-ba40-fd3daaabdf23`\n\nor
+      specify only one recipe UUID:\n`e20da1a3-313c-434a-9d43-7268b12fee08`  "
+    is_required: true
+    summary: ""
+    title: Recipe UUID
+  recipe_uuid: ""
+- adb_serial_port: ""
+  opts:
+    description: "If `adb_serial_port` option is set,  \nthe instance will be connected
+      to ADB on localhost:`adb_serial_port`.  \nOtherwise it will be connected to
+      a port given by your system,  \n`gmsaas instances list` to know which one.\n\nFormat:\nYou
+      can specify several ADB Serial ports. ADB Serial ports are separated with a
+      comma.\n\nFor example:\n`4321,4322,4323`"
+    summary: ""
+    title: ADB serial port
+outputs:
+- GMCLOUD_SAAS_INSTANCE_UUID: null
+  opts:
+    description: |
+      Return the UUID list of started and connected instances.
```

Comment:
I suggest highlighting the format of the list or the separator, this way users will know which step and how can receive this output. The same is true for every list type inputs and outputs.

Reply:
Hi @godrei, Thanks for the review. I made changes , so  will close this PR and open a new one.